slo_generator/backends/cloud_service_monitoring.py

```diff
@@ -342,7 +356,8 @@ def create_slo(self, window, slo_config):
         return SSM.to_json(slo)
 
     @staticmethod
-    def build_slo(window, slo_config):  # pylint: disable=R0912,R0915
+    def build_slo(window: int,
+                  slo_config: dict) -> dict:  # pylint: disable=R0912,R0915
```

Comment:
Move the `# pylint: disable` directive to the line above and `make lint` should succeed.

I just discovered that directives such as this one must be on the same line as `def`.

Reply:
I have added those options to `.pylintrc`.